follower/consensus_follower.go

```diff
@@ -148,9 +147,9 @@ func getBaseOptions(config *Config) []cmd.Option {
 	return options
 }
 
-func buildAccessNode(accessNodeOptions []access.Option) (*access.UnstakedAccessNodeBuilder, error) {
-	anb := access.FlowAccessNode(accessNodeOptions...)
-	nodeBuilder := access.NewUnstakedAccessNodeBuilder(anb)
+func buildObserver(accessNodeOptions []FollowerOption) (*ObserverServiceBuilder, error) {
+	anb := FlowObserverService(accessNodeOptions...)
+	nodeBuilder := WithSkipValidations(anb)
```

Comment:
Is there a situation where we want this setting disabled? Also, why manage this setting differently in the observer and follower?